CONTRIBUTING.md

```diff
@@ -0,0 +1,206 @@
+# Contributing to OurPromise 🤝🏻
+
+🎊If you're reading this, you're probably creating a Pull Request or planning to do so and that's great!🎊
+
+This project is actively looking for contributors!
+The following is a set of guidelines for contributing to this project, and you can see the live instance of the project [here](https://ourpromise.web.app/).
+
+These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+**Table Of Contents**
+
+[Code of Conduct 🗒](#code-of-conduct-🗒)
+
+[Before You Contribute 🏁](#before-you-contribute-🏁)
+
+[Local Setup 💻](#local-setup-💻)
+
+[How Can You Contribute 📝](#how-can-you-contribute-📝)
+
+- [Reporting Bugs 🐛](#reporting-bugs-🐛)
+- [Suggesting Enhancements ✨](#suggesting-enhancements-✨)
+- [First Code Contribution 🥇](#first-code-contribution-🥇)
+- [Pull Request 🥳](#pull-request-🥳)
+
+[Getting Your Changes Reviewed 📝](#getting-your-changes-reviewed-📝)
+
+[Pull Request Reviewers Guide 🦮](#pull-request-reviewers-guide-🦮)
+
+## Code of Conduct 🗒
+
+Everyone participating in this project is requested to practice decent behaviour. Help others, share knowledge and grow your knowledge!
+
+## Before You Contribute 🏁
+
+Welcome to [OurPromise](https://ourpromise.web.app/). Before you send the pull request make sure that you read the whole guidelines.
+If you have any doubts relating to an issue, comment under it and tag members if needed.
+Please don't file an issue to ask a question.
+
+## Local Setup 💻
+
+**Step 0:** Clone the forked repo and cd into it
+
+```
+git clone https://github.com/<Your username>/OurPromise.git
+cd OurPromise/
+```
+
+**Step 1:** To install all the required packages run `npm install`.
+
+**Step 2:** Run `npm start` to start the app. After successful compilation the app would be running in `http://localhost:3000/`.
+
+**Note:** If the process was successful, you would recieve the following in your terminal
+
+```
+Compiled with warnings.
+
+[eslint]
+src\components\pages\GraduateDetails.tsx
+  Line 122:9:  Unexpected console statement  no-console
+
+src\components\pages\Lecturers.tsx
+  Line 160:9:  Unexpected console statement  no-console
+
+src\components\pages\auth\Create.tsx
+  Line 92:5:  Unexpected alert  no-alert
+xt-line to the line before.
+
+WARNING in [eslint]
+src\components\pages\GraduateDetails.tsx
+  Line 122:9:  Unexpected console statement  no-console
+
+src\components\pages\Lecturers.tsx
+  Line 160:9:  Unexpected console statement  no-console
+
+src\components\pages\auth\Create.tsx
+  Line 92:5:  Unexpected alert  no-alert
+
+src\components\pages\auth\Login.tsx  Line 123:5:  Unexpected alert  no-alert
+
+src\components\pages\graduates\Graduates.tsx
+  Line 150:9:  Unexpected console statement  no-console
+
+webpack compiled with 1 warning
+No issues found.
+```
```

Comment:
[Discussion]
I think this is too lengthy to list all warnings here.
What do you think? 👇 

```suggestion
**Note:** If the process was successful, there might be ESLint warnings on your terminal. There are meant to be refactor/fixed gradually
```

Reference: https://dev.to/thawkin3/eslint-warnings-are-an-anti-pattern-33np

Reply:
Tbh I also thought the same! So I have 3 solutions! 
* Keep a screenshot of the terminal and give it a specific size so it would occupy less space.
* Instead of writing the whole terminal we can only write,  `No issues found.` I have seen this, in one of the projects where I worked before, don't remember which was it. 
* Remove it!
Personally, I would prefer 2 because It's clear, and have seen it in other projects too!
So what do you think?

Reply:
ok, we can with the 2nd option

Reply:
@aswin2108 , please re-request review after you have ammeded ya. Thanks

![image](https://user-images.githubusercontent.com/34387770/209433413-e16684df-f9af-48e1-b140-86246d6b7664.png)